inference-engine/src/mkldnn_plugin/utils/README.md

```diff
@@ -71,3 +71,21 @@ Example:
 ```sh
     OV_CPU_BLOB_DUMP_NODE_NAME=".+" binary ...
 ```
+
+## Graph serialization
+The functionality allows to serialize execution graph using environment variable:
+```sh
+    OV_CPU_EXEC_GRAPH_PATH=<path> binary ...
+```
+
+Possible serialization options:
+* cout
+
+    Serialize to console output
+* .xml
+
+    Serialize graph into .xml and .bin files. Can be opened using, for example, *netron* app
+* .dot
+
+    TBD
```

Comment:
Can we update this description as well?

Reply:
Could you please clarify, what exactly should we update here?
Do you mean <path>.dot?

Reply:
Discussed, will be updated